packages/react/src/wallet/useWalletLogin.ts

```diff
@@ -4,15 +4,15 @@ import {
   WalletConnectionError,
 } from '@lens-protocol/domain/entities';
 import { WalletLoginResult } from '@lens-protocol/domain/use-cases/wallets';
-import { Signer } from 'ethers';
+import { EthereumAddress } from '@lens-protocol/shared-kernel';
 
 import { Operation, useOperation } from '../helpers/operations';
 import { useWalletLoginController } from './adapters/useWalletLoginController';
 
 export type WalletLoginOperation = Operation<
   WalletLoginResult,
   PendingSigningRequestError | WalletConnectionError | UserRejectedError,
-  [Signer, string?]
+  [string, string?]
```

Comment:
Not advocating for the fact `useWalletLogin` used to accept an ether.js `Signer`. In hindsight probably it was not a good idea.

The only thing is that this change is a breaking change fro `@lens-protocol/react` (and inevitably `@lens-protocol/react-web` package). Did you put any thought on how to move forward?